### PR TITLE
feat: improvement of package list

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Visualize package data using a list instead of a table. This allows you to view the package data without any problems even in small window sizes.

**This PR depends on the following PR(s).**
- #198

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/package/package.js [hunk 1]
```diff
@@ -93,21 +74,24 @@ async function setPackagesList(instPath, minorUpdate = false) {
...
+  // sort buttons
+  if (!packagesSort.hasChildNodes()) {
+    Array.from(columns.entries())
+      .filter(([n, s]) => ['name', 'developer'].includes(s))
+      .forEach(([i, columnName]) => {
+        const sortBtn = document
+          .getElementById('sort-template')
+          .cloneNode(true);
+        sortBtn.removeAttribute('id');
+        sortBtn.setAttribute('data-sort', columnName);
```
Comment (hal-shu-sato) [inline src/package/package.js:91]: And while we're at it, let's change this, too.
```suggestion
        sortBtn.dataset.sort = columnName;
```

**Review by hal-shu-sato — COMMENTED** [review]
It's very good.
I have one noticed point not related to this feature.
It is better to fix it at this opportunity.

**Comment by mitosagi** [comment]
I fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Thanks.

**Pull request merged**